news.md

```diff
@@ -6,10 +6,11 @@ permalink: /news/
 
   {% for page in site.posts %}
   <article>
-    <h2>
-      <a href="{{ page.url }}">{{ page.title }}</a>
+    <h3>
+      {{ page.title }}
```

Comment:
Why don't we have the link to the individual page now?

Reply:
I didn't think it was necessary. Also, it's not great accessibility-wise if a heading also serves as a link. There are other ways to solve that problem, though. Did we talk about this and I forgot? In the long run, I think it would be good to roll things into archive-like sub-pages, like all of 2016 news, for example.

Reply:
Sounds good.